global/global.go

```diff
@@ -0,0 +1,57 @@
+package global
```

Comment:
The license is missing

Reply:
sorry, solved in 18c38772a02471e367ee7c0f5f553bd6e3f1ce44